### PR TITLE
Sscs 10245 send epimms id to hmc new

### PR DESCRIPTION
**Pull request by suetlaiHMCTS**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by wmarshid — DISMISSED** [review]

**Review by bensonlievoco — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/sscs/validator/EnumPatternValidatorTest.java [hunk 1]
```diff
@@ -29,31 +29,6 @@ public void setUp() {
         validator = factory.getValidator();
     }
 
-    @ParameterizedTest
```
Comment (suetlaiHMCTS) [inline src/test/java/uk/gov/hmcts/reform/sscs/validator/EnumPatternValidatorTest.java:32]: No more enumPattern in HearingLocations so I remove those tests

File: src/main/java/uk/gov/hmcts/reform/sscs/model/single/hearing/HearingLocations.java [hunk 2]
```diff
@@ -5,17 +5,15 @@
...
 import lombok.Data;
 import lombok.NoArgsConstructor;
-import uk.gov.hmcts.reform.sscs.validator.EnumPattern;
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class HearingLocations {
```
Comment (dattipoeHMCTS) [inline src/main/java/uk/gov/hmcts/reform/sscs/model/single/hearing/HearingLocations.java:14]: Could you please add the @JsonIgnoreProperties(ignoreUnknown = true) annotation?

**Review by dattipoeHMCTS — DISMISSED** [review]

**Review by bensonlievoco — DISMISSED** [review]

**Review by wmarshid — APPROVED** [review]

**Review by wmarshid — APPROVED** [review]

**Review by bensonlievoco — DISMISSED** [review]

**Review by MohammedHMCTS — DISMISSED** [review]

**Review by MohammedHMCTS — DISMISSED** [review]

**Review by nickwest1 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java [hunk 3]
```diff
@@ -155,14 +155,14 @@ public static boolean shouldBeHearingsInWelshFlag() {
...
-        // locationType - from reference data - processing venue to venue type/epims
-        // locationId - epims
-        // manual over-ride e.g. if a judge wants to change venue
-        // if paper case - display all venues in that region
-        // locations where there is more than one venue
-        // Normally one location, but can be two in some cities.
-        // TODO Implementation to be done by SSCS-10245 - work out what venues to choose and get epims/locationType info from Reference Data
-        return new ArrayList<>();
+        HearingLocations hearingLocations = new HearingLocations();
+        hearingLocations.setLocationId(caseManagementLocation.getBaseLocation());
```
[the remark this answers is not in the document]
Comment (suetlaiHMCTS) [reply src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java:159]: @dattipoeHMCTS May i know why.

[the remark this answers is not in the document]
Comment (wmarshid) [reply src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java:159]: @dattipoeHMCTS can we confirm this requirement asap please

**Review by dattipoeHMCTS — APPROVED** [review]
Fix for hearing locations to come in subsequent PR.

**Review by nickwest1 — APPROVED** [review]

**Pull request merged**